src/main/java/beans/menus/EntityListResponse.java

```diff
@@ -230,7 +230,13 @@ private void processStyles(Detail detail) {
     }
 
     private void processActions(SessionWrapper session) {
-        Vector<Action> actions = session.getDetail(((EntityDatum) session.getNeededDatum()).getShortDetail()).getCustomActions(session.getEvaluationContext());
+        EntityDatum datum = (EntityDatum) session.getNeededDatum();
+        // HACKY! Uses the fact that case claim results will always be in a "result" instance
```

Comment:
@proteusvacuum Sadly Mr. Vacuum that SessionDatum isn't available anymore - we've already proceeded on to the next step at this point, so we only know that we're on an entity list and not about the genesis of the nodeset that we're displaying. You did help me remember, however, that we *do* have access to the frame stack and specifically the previous step we took and so can check for the type of that step